pages/price-feeds/fetch-price-updates.mdx

```diff
@@ -0,0 +1,76 @@
+import { Callout } from "nextra/components";
+
+# How To Fetch Price Updates
+
+The following guides explain how to fetch price updates to update Pyth price feeds on-chain.
+Price updates can be submitted to the Pyth Price Feeds contract to update the on-chain price.
+These price updates are fetched from [Hermes](/price-feeds/how-pyth-works/hermes) which are verified on-chain.
+Please see [What is a Pull Oracle?](/price-feeds/pull-updates) to learn more.
+
+There are three different ways to fetch price updates:
+
+1. [REST API](#rest-api)
+1. [SDK](#sdk)
+1. [Websocket](#websocket)
+
+<Callout emoji="ℹ️">
+  You need to pass the Pyth price feed's ID to request the price updates. These
+  IDs are unique identifiers for price feeds (e.g., BTC/USD). \ The complete
+  list of Pyth price feed IDs is available at
+  https://pyth.network/developers/price-feed-ids
+</Callout>
+
+## REST API
+
+Hermes exposes APIs to fetch the latest price updates.
+Consult the [Hermes API documentation](https://hermes.pyth.network/docs/#/) for more information.
+
+Here is an example of retrieving the latest price update of the BTC/USD price feed:
+
+```bash
+curl -X 'GET' \
+  'https://hermes.pyth.network/v2/updates/price/latest?ids%5B%5D=0xe62df6c8b4a85fe1a67db44dc12de5db330f7ac66b72dc658afedf0f4a415b43' \
+  -H 'accept: application/json'
```

Comment:
do you really need the -X 'GET' and the -H bits? the command works for me without either of those:

```
curl https://hermes.pyth.network/v2/updates/price/latest?ids[]=0xe62df6c8b4a85fe1a67db44dc12de5db330f7ac66b72dc658afedf0f4a415b43
```

Reply:
We can exclude the headers, but it's a good practice to mention call type 'GET'